### PR TITLE
Cypress tests

### PR DESCRIPTION
**Pull request by avanti-em**

### CODE REVIEW

File: .github/workflows/webviz-subsurface-components.yml
Comment (anderskiaer) [inline .github/workflows/webviz-subsurface-components.yml:101]: Should we turn this into a SHA? I.e. instead of `@versionnumber` we use `@sha`

https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions#using-third-party-actions

Comment (anders-kiaer) [inline .github/workflows/webviz-subsurface-components.yml:101]: Should we turn this into a SHA? I.e. instead of `@versionnumber` we use `@sha`

https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions#using-third-party-actions

Comment (avanti-em) [reply .github/workflows/webviz-subsurface-components.yml]: Hi @anders-kiaer , according to the official documentation of Github:
[](https://github.com/actions/upload-artifact#retention-period)

Artifacts are retained for 90 days by default. We can specify a shorter retention period using the "retention-days" input.
for example we will keep files only for 5 days. So that it does not put stress on our storage capacity.
The retention period for a pull request restarts each time someone pushes a new commit to the pull request.

Right now, we store images only if a test fails. And videos are always recorded despite of pass or failure.
So we can stop recording videos to further save our storage

Comment (anders-kiaer) [reply .github/workflows/webviz-subsurface-components.yml]: Sounds great @avanti-em  - thanks for the information. Then as you say, storage should not be a problem.

The new actions included in this PR are approved, it might be that the PR needs a new commit or something to restart the workflow/CI?

Comment (avanti-em) [reply .github/workflows/webviz-subsurface-components.yml]: I have made the updates, and triggered the pipeline

**Review by anders-kiaer — APPROVED** [review]
LGTM when CI is 🟢👍

**Comment by hkfb** [comment]
Blocked by #900

**Comment by hkfb** [comment]
@avanti-em looks like the reference screenshots need to be updated?

**Comment by avanti-em** [comment]
> @avanti-em looks like the reference screenshots need to be updated?

hi @hkfb , I have updated the screenshots. The tests are also passing.

**Pull request merged**
